core/client/src/runtime_api/mod.rs

```diff
@@ -73,6 +72,30 @@ pub trait CallApiAt<Block: BlockT> {
 		changes: &mut OverlayedChanges,
 		initialised_block: &mut Option<BlockId<Block>>,
 	) -> error::Result<Vec<u8>>;
+
+	/// Call the given api function with strong arguments at the given block
+	/// and returns the decoded result.
+	fn call_api_at_strong<In: Encode, Out: Decode>(
```

Comment:
This trait should just be used by the runtime api stuff and that does not require this function. Please remove it.